jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2740,6 +2741,16 @@ private String toResourceName(String resName) {
  */
 @Override
 public String toString() {
+	String localToStringName = toStringName;
```

Comment:
I'm not a fan of the names `toStringName` or `localToStringName` (the suffix "Name" seems inappropriate).
Other caches are named `cachedXxxx`: I suggest the field should be named `cachedToString` and the local could be just `result`.

Reply:
done